finagle-http/src/main/scala/com/twitter/finagle/http/Status.scala

```diff
@@ -1,20 +1,20 @@
 package com.twitter.finagle.http
 
 /**
- * Represents an HTTP status code.
- *
- * The set of commonly known HTTP status codes have an associated reason phrase
- * (see `reasons`). We don't provide a way to set the reason phrase because:
- *
- * - it simplifies construction (users only supply the code)
- * - it avoids the need to validate user-defined reason phrases
- * - it omits the possibility of statuses with duplicate reason phrases
- *
- * The only downside is that we lose the ability to create custom statuses with
- * "vanity" reason phrases, but this should be tolerable.
- *
- * For Java-friendly enums, see [[com.twitter.finagle.http.Statuses]].
- */
+  * Represents an HTTP status code.
```

Comment:
Yep, we adhere to the comment style here: https://twitter.github.io/effectivescala/#Formatting-Comments